pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -298,42 +279,38 @@ def __call__(self, params, t):
 
     def __add__(self, other):
         if isinstance(other, HardwareHamiltonian):
-            # Update coeffs, obs and hardware attributes
-            if self.register is not None:
-                if other.register is not None:
-                    raise ValueError("We cannot add two Hamiltonians with an interaction term!")
-                if not self.wires.contains_wires(other.wires):
```

Comment:
Are we dropping the non-matching wires warning, or adding it in a future PR?

Reply:
Hm good point, it is not so easy though because the `settings` objects dont have wires actually 🤔 something we can consider adding later?